runtime/package.json

```diff
@@ -39,8 +39,8 @@
     "webpack-dev-server": "webpack-dev-server",
     "webpack": "webpack",
     "test:unit": "karma start tests/karma.conf.js",
-    "test:func": "./src/tests/functionalTests/run_ts_functional_tests.sh",
-    "test:funcsmoke": "./src/tests/functionalTests/run_ts_functional_tests.sh smokeTest"
+    "test:func": "./tests/run_functional_tests.sh",
+    "test:funcsmoke": "./tests/run_functional_tests.sh smokeTest"
```

Comment:
I don't see `run_functional_tests.sh` file in `tests` directory

Reply:
Removed the entire scripts here. The tests should be run using `//scripts/run-functests.sh`. The entire runtime directory will be deprecated soon.